component.json

```diff
@@ -5,11 +5,47 @@
   "docsUrl":"https://github.com/elasticio/postgresql-component",
   "credentials": {
     "fields": {
-      "conString": {
-        "label": "Connection String",
+      "protocol": {
```

Comment:
Do we need to maintain backwards compatibility with `conString` at all?

Reply:
Yeah we do. I've fixed it